client/package.json

```diff
@@ -41,7 +41,7 @@
     "lint:js": "eslint .",
     "lint:css": "stylelint '**/*.{css,scss,sass}'",
     "build": "NODE_ENV=production BABEL_ENV=production webpack -p --display-error-details --config webpack.prod.js",
-    "test": "jest",
+    "test": "TZ=UTC-6 jest",
```

Comment:
Here, I am setting the time zone to be CST for our tests.  An alternative would be to use the jest global-setup.js outlined here:  https://stackoverflow.com/questions/56261381/how-do-i-set-a-timezone-in-my-jest-config?answertab=votes#tab-top

Reply:
@nathanfranklin Since we're only relying on this env var for testing, this seems fine to me.